### ISSUE TITLE
Request: macOS/iOS "Metal" port

### ISSUE THREAD

**Issue by ronaaron** [Immediate-Mode-UI/Nuklear#43]
Being able to have a version run using the native "Metal" API on macOS/iOS would be extremely helpful, as I'm migrating [my tool](https://8th-dev.com/) from using the JUCE library, and I would greatly prefer to not rely on "glfw".

If someone has already done something along these lines, please inform me; Google didn't turn up anything useful.

**Comment by dumblob** [comment]
Maybe there is some Metal backend for ImGui to get inspiration from (perhaps https://github.com/floooh/sokol-samples/blob/master/metal/imgui-metal.cc ?).

**Comment by ronaaron** [comment]
Yes, Dear ImGui does have a Metal backend; but I was hoping someone had already either ported that or had a similar bit of working code for Nuklear.

**Comment by ronaaron** [comment]
I'm posting about my progress porting from JUCE to Nuklear [on the 8th forum](https://8th-dev.com/forum/index.php/topic,1977.0.html), if you're interested
